app/assets/javascripts/surveys.js

```diff
@@ -106,15 +106,15 @@
     },
 
     userCompletedTransaction: function() {
-      var currentURL = window.location.pathname;
+      var currentPath = userSurveys.currentUrl();
```

Comment:
`/path-to-resource`, so I'll fix the helper method to be currentPath rather than URL too. Good spot.